src/main/java/io/dropwizard/redis/HostAndPort.java

```diff
@@ -7,15 +7,15 @@
 
 import javax.validation.constraints.NotNull;
 
-public class Node {
+public class HostAndPort {
```

Comment:
Guava’s HostAndPort?

Reply:
Good point, wasn't sure if the Guava Jackson module supported serialization for that, but it does, so I'll switch.